content/docs/authorization/Backup and Restore/helm/_index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Helm
+linktitle: Helm
+description: >
+  Dell Technologies (Dell) Container Storage Modules (CSM) for Authorization Helm backup and restore
+---
+
```

Comment:
Instead of repeating "In the deployment of Authorization that you want to restore" for each step, can we put this snippet before listing the steps for Roles?

Reply:
The only issue with that is that step 1 is ran in the original deployment to save the resource. Would having sections for steps to execute on the original deployment vs the new deployment work?

Reply:
Updated with sections for the different deployments.